020-LinuxFundamentals/Student/resources/linux-history.md

```diff
@@ -0,0 +1,67 @@
+## Linux History
+
+Linux borrows heavily from the UNIX operating system because it was written to be a free and open source version of UNIX. Files are stored in a hierarchical filesystem, with the top node of the system being root or simply "/". 
+
+Whenever possible, Linux makes its components available via files or objects that look like files. Processes, devices, and network sockets are all represented by file-like objects, and can often be worked with using the same utilities used for regular files.
+
+Linux is a fully multitasking (a method where multiple tasks are performed during the same period of time), multiuser operating system, with built-in networking and service processes known as daemons in the UNIX world. Linux was inspired by UNIX, but it is not UNIX.
+
+**So, what is a Linux distribution and how does it relate to the Linux kernel?**
+
+As illustrated below, the [Linux kernel](http://www.kernel.org) is the core of a computer operating system. A full **Linux distribution** consists of the kernel plus a number of other software tools for file-related operations, user management, and software package management. Each of these tools provides a small part of the complete system. Each tool is often its own separate project, with its own developers working to perfect that piece of the system.
+
+<img align="center" src="./images/linux-intro.png"/>
```

Comment:
All images on this page are transparent and designed to be shown on a white background.  When using GitHub's dark mode AND the theme on our student website, these images are not clearly visible.  Remove the transparency from the images and just use a white backround.